docs/content/templates/functions.md

```diff
@@ -740,6 +746,23 @@ This can be useful if you want to use Gravatar for generating a unique avatar:
 <!-- returns the string "c8b5b0e33d408246e30f53e32b8f7627a7a649d4" -->
 ```
 
+## Internationalization
+
+### i18n
+
+This translates a piece of content based on your `i18n/en-US.yaml`
+(and friends) files. You can use the
+[go-i18n](https://github.com/nicksnyder/go-i18n) tools to manage your
+translations.  The translations can exist in both the theme and at the
+root of your repository.
+
+e.g.: `{{ i18n "translation_id" }}`
+
+
+### T
+
+`T` is an alias to `i18n`. E.g. `{{ T "translation_id" }}`.
+>>>>>>> Add multilingual support in Hugo
```

Comment:
Line 765 can just be deleted. This todo is done (Yay!). Maybe we could add a link to /content/multilingual/#translation-of-strings to clarify.